pkg/psmdb/container.go

```diff
@@ -113,6 +113,14 @@ func container(ctx context.Context, cr *api.PerconaServerMongoDB, replset *api.R
 				Name:  "MONGODB_REPLSET",
 				Value: replset.Name,
 			},
+			{
+				Name:  "REPLSET_EXPOSED",
+				Value: strconv.FormatBool(replset.Expose.Enabled),
+			},
+			{
+				Name:  "REPLSET_EXPOSED_TYPE",
+				Value: string(replset.Expose.ExposeType),
+			},
```

Comment:
do we want to restart pods if backups are not enabled? this container is `mongod`, we should put these into `backup-agent` container.

Reply:
Yes, I just realized that my self.

Reply:
Done: [f91c1a9](https://github.com/percona/percona-server-mongodb-operator/pull/1152/commits/f91c1a9330491b8965eb3b7091c7209a105947df)